### PR TITLE
More Python 2 / 3 compatibility adjustments

### PR DESCRIPTION
**Pull request by davilima6**

### CODE REVIEW

File: plone/app/caching/browser/edit.py [hunk 1]
```diff
@@ -60,13 +60,13 @@ class EditForm(form.Form):
...
-        operation,
-        rulesetName=None,
-        ruleset=None
+            self,
+            context,
+            request,
+            operationName,
+            operation,
+            rulesetName=None,
+            ruleset=None,
```
Comment (jensens) [inline plone/app/caching/browser/edit.py]: Why two indent levels here?

File: plone/app/caching/operations/utils.py [hunk 2]
```diff
@@ -49,14 +49,14 @@
...
-    lastModified=None,
-    vary=None
+        published,
+        request,
+        response,
+        maxage=None,
+        smaxage=None,
+        etag=None,
+        lastModified=None,
+        vary=None,
```
Comment (jensens) [inline plone/app/caching/operations/utils.py]: Same as above

File: plone/app/caching/operations/utils.py [hunk 3]
```diff
@@ -144,13 +144,13 @@ def cacheInBrowser(published, request, response, etag=None, lastModified=None):
...
-    etag=None,
-    lastModified=None,
-    vary=None
+        published,
+        request,
+        response,
+        smaxage,
+        etag=None,
+        lastModified=None,
+        vary=None,
```
Comment (jensens) [inline plone/app/caching/operations/utils.py]: And here

File: plone/app/caching/operations/utils.py [hunk 4]
```diff
@@ -182,14 +182,14 @@ def cacheInProxy(
...
-    lastModified=None,
-    vary=None
+        published,
+        request,
+        response,
+        maxage,
+        smaxage=None,
+        etag=None,
+        lastModified=None,
+        vary=None,
```
Comment (jensens) [inline plone/app/caching/operations/utils.py]: Again

File: plone/app/caching/operations/utils.py [hunk 5]
```diff
@@ -744,11 +744,11 @@ def getRAMCacheKey(request, etag=None, lastModified=None):
...
-    request,
-    response,
-    result,
-    globalKey=PAGE_CACHE_KEY,
-    annotationsKey=PAGE_CACHE_ANNOTATION_KEY
+        request,
+        response,
+        result,
+        globalKey=PAGE_CACHE_KEY,
+        annotationsKey=PAGE_CACHE_ANNOTATION_KEY,
```
Comment (jensens) [inline plone/app/caching/operations/utils.py]: And so on, see below

**Comment by davilima6** [comment]
@jensens, thanks! Fixed now. It seems my local linter gave a found positive for those.

**Pull request merged**